app/controllers/hub/sites_controller.rb

```diff
@@ -22,19 +22,17 @@ def create
     end
 
     def edit
-      @site = @vita_partner
-      @routing_form = ZipCodeRoutingForm.new(@vita_partner)
-      @source_params_form = SourceParamsForm.new(@vita_partner)
+      @routing_form = ZipCodeRoutingForm.new(@site)
+      @source_params_form = SourceParamsForm.new(@site)
       @show_unique_links = true
     end
 
     def update
-      if @vita_partner.update(vita_partner_params)
+      if @site.update(site_params)
```

Comment:
I wonder if `if @site.save` has the same effect -- does cancancan update the attributes for us? Not a blocker.